Windows/PlayerWindow.py

```diff
@@ -165,8 +165,10 @@ def trigger_play(self):
                     tm.sleep(1/self._speed)
                     if self.running:
                         self._iterator += 1
+                        c = 0
```

Comment:
Is it necessary?

Reply:
Yeah its a leftover, i have removed it in the next pull request